dashboard/app/views/levels/editors/fields/_poetry_fields.html.haml

```diff
@@ -19,3 +19,13 @@
   #defaultPoem{class: ('collapse' if @level.standalone_app_name == 'poetry')}
     = f.label :default_poem, 'Default Poem'
     = f.select :default_poem, options_for_select(@level.class.poems_for_subtype(@level.standalone_app_name), @level.default_poem)
+
+    = f.label :dropdown_poems, 'Available Poems'
```

Comment:
If we tell levelbuilders that these are the "available poems", maybe that's a good indicator that we should also use that term instead of `dropdown_poems`.